src/common/utils/compare.ts

```diff
@@ -258,3 +273,78 @@ export function isCollegeCounty(location: Location) {
   const countyPopulation = location.population;
   return ftEnrollment && ftEnrollment / countyPopulation > threshold;
 }
+
+// For sharing:
+
+export function getShareQuote(
+  sorter: Metric,
+  countyTypeToView: MetroFilter,
+  sliderValue: GeoScopeFilter,
+  totalLocations: number,
+  sortDescending: boolean,
+  currentLocation?: RankedLocationSummary,
+  sortByPopulation?: boolean,
+  isHomepage?: boolean,
+  viewAllCounties?: boolean,
+  stateName?: string,
+): string {
+  const sliderValueShareCopy: any = {
+    [GeoScopeFilter.NEARBY]: 'nearby',
+    [GeoScopeFilter.STATE]: stateName,
+    [GeoScopeFilter.COUNTRY]: 'USA',
+  };
+
+  const homepageShareCopy = `Compare all USA ${
+    viewAllCounties
+      ? `${getMetroPrefixCopy(countyTypeToView)} counties`
+      : 'states'
+  } by their local COVID metrics at CovidActNow.`;
+
+  const stateShareCopy = `Compare COVID metrics between ${getMetroPrefixCopy(
+    countyTypeToView,
+    true,
+  )} counties in ${stateName} with @CovidActNow.`;
+
+  const hasValidRank =
+    currentLocation &&
+    currentLocation.rank !== 0 &&
+    currentLocation.metricsInfo.metrics[sorter]?.value;
```

Comment:
This breaks if you share counties with `0` for a metric value, I think?

e.g. https://covid-projections-git-casulin-compare-share-quotes.covidactnow.vercel.app/us/wa/county/lincoln_county

Reply:
Oo yep. Fixed